sdk/personalizer/ai-personalizer-rest/samples-dev/multiSlotRankActionsAndrewardEvents.ts

```diff
@@ -0,0 +1,103 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+/**
+ * @summary Demonstrates the use of a Personalizer client to rank actions for multiple slots and reward the presented action.
+ */
+import Personalizer, {
+  isUnexpected,
+  MultiSlotRankRequest,
+  PersonalizerErrorOutput,
+  RankableAction,
+  SlotRequest,
+  SlotResponseOutput,
+} from "@azure-rest/ai-personalizer";
+
+// Load the .env file if it exists
+import * as dotenv from "dotenv";
+dotenv.config();
+
+async function main() {
+  const endpoint = process.env.PERSONALIZER_ENDPOINT || "<endpoint>";
+  const key = process.env.PERSONALIZER_API_KEY || "<test-key>";
+
+  const client = Personalizer(endpoint, { key: key });
+
+  const request: MultiSlotRankRequest = {
+    slots: getSlots(),
+    actions: getActions(),
+    contextFeatures: getContextFeatures(),
+  };
+
+  log("Sending multi-slot rank request");
```

Comment:
why not just call `console.log` directly?

Reply:
If any one wants to copy this code as is and change how to log in their code, they can do it in just one place instead of every where.